src/libs/validationHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response, NextFunction } from 'express';
+export default (config) => (req: Request,res: Response,next: NextFunction) => {
+    const errors = [];
+    Object.keys(config).forEach((key) => {
+        const i = 0;
+        const keys = config[key];
+        const locations = keys.in[i];
+        let request = req[locations][key];
+        const regex = keys.regex;
+        if ((keys.required) && !(request)) {
```

Comment:
`!(request) ` will return true for number 0, which is a valid value for number type